tsl/test/isolation/expected/compression_dml_iso.out

```diff
@@ -187,7 +190,7 @@ step DEL1: BEGIN; DELETE from ts_device_table WHERE location = 200;
 step UPDrr: BEGIN TRANSACTION ISOLATION LEVEL REPEATABLE READ; UPDATE ts_device_table SET value = 4 WHERE location = 200; <waiting ...>
 step DELc: COMMIT;
 step UPDrr: <... completed>
-ERROR:  could not serialize access due to concurrent update
+ERROR:  unexpected tuple lock status
```

Comment:
why do these messages change?

Reply:
I see that return status from table_tuple_delete() was handled similarly in many different parts of the code. So following the same.

Previously for every return status from table_tuple_delete() we were reporting a different error. Now instead we report 1 single error message.

Reply:
Having distinct error messages for different situation is extremely useful when debugging. Throwing the same generic error message makes it harder to debug because it blurs what is actually happening.

Reply:
Ok. Fixed.